tst/teststandard/grpauto.tst

```diff
@@ -63,8 +63,6 @@ gap> H:=PcGroupCode(
 > 2361539190251190837914203692538788582942691934753704906691,20000);;
 gap> IsomorphismGroups(G,H);
 fail
-gap> IsomorphismGroups(G,PcGroupCode(CodePcGroup(G),Size(G)))=fail;
-false
```

Comment:
Actually, in my attempts the command above passed the test with `-o 1g` setting, though it probably took too long. However, another call to `IsomorphismGroups` below caused running out of memory limits:
```
@@ -107,8 +107,6 @@ gap> H:=PcGroupCode(18738408935359210460657231881739776911013615108923450662760,
 > 21952);;
 gap> IsomorphismGroups(G,H);
 fail
-gap> IsomorphismGroups(G,PcGroupCode(CodePcGroup(G),Size(G)))=fail;
-false
 
 gap> G:=PcGroupCode(49879636940338958988550512603242447645113136854728735,
 > 15552);;
```

so maybe we need to remove it as well (I am happy to move it to benchmarks - just looked at them in #966).